### DISCUSSION TITLE
toolbar

### DISCUSSION THREAD

**Discussion by sm-hejazi** [remarkjs/remark#1144] [Q&A] [answered]
**How to add toolbar** 
like this image:

![example](https://user-images.githubusercontent.com/1094760/33604369-0e5b5c50-d9f9-11e7-9608-43390260bf3e.png)

**Comment by wooorm** [comment, marked as answer]
That’s outside of markdown.
You can make such a toolbar with any framework, react, preact, vue, or with normal JavaScript.
Then you figure out where the user’s selection or caret is.
And then, depending on the command, do something.
For a “bold” button, you’d wrap the selection with `**` and `**` for example.
